### PR TITLE
Yashwant

### PR DESCRIPTION
**Pull request by Yashwant-0108**
will you plz review my work and kindly let me know if changes are there

### CODE REVIEW

File: src/Components/Home.js [hunk 1]
```diff
@@ -79,20 +85,20 @@ export default function Example() {
...
-      ) : (
-        <SignupForm toggleLogin={toggleForm} />
-      )}
-      </div>
+              <div className="text-center">
+                 <h1 className="text-4xl font-bold tracking-tight text-gray-900 sm:text-4xl">
+                     One platform to streamline all workflows
+                  </h1>
+                  <p className="mt-2 text-lg leading-8 text-gray-600">
+                  Your all-in-one platform to manage projects, organize work, enhance collaboration, and accelerate execution across all departments.
```
Comment (anupkore) [inline src/Components/Home.js:93]: Please Format the code in proper standard format and remove black spaces and comments

File: src/Components/Navbar.js [hunk 2]
```diff
@@ -23,6 +23,10 @@ export default function Navbar()
         <div className="flex lg:flex-1 text-lg font-semibold">
           <Link to={"/"}><img src='/Images/jira-low-resolution-logo-color-on-transparent-background.png' alt='PMT LOGO' height="30px" width="30px"></img></Link>
         </div>
+        {/* <legend style={linkStyle} className="text-lg font-semibold leading-6 text-gray-900 lg:justify-end pointer-cursor" onClick={handleLogout}>
+            Logout <span aria-hidden="true">&rarr;</span>
```
Comment (anupkore) [inline src/Components/Navbar.js:27]: Please remove this commented part if not required in future

File: src/Components/ProjectCards.js
Comment (anupkore) [inline src/Components/ProjectCards.js:1]: This component is no more required in our project..you can delete this

File: src/Components/SignupForm.js
Comment (anupkore) [inline src/Components/SignupForm.js:1]: This component is no more required in our project...you can delete this

**Review by anupkore — CHANGES_REQUESTED** [review]
Please do the needful changes

File: src/Components/ProjectCards.js
Comment (Yashwant-0108) [reply src/Components/ProjectCards.js:1]: if not required i will remove it

File: src/Components/SignupForm.js
Comment (Yashwant-0108) [reply src/Components/SignupForm.js:1]: we need this component for the future purpose

File: src/Components/Navbar.js (hunk 2 above)
Comment (Yashwant-0108) [reply src/Components/Navbar.js]: i have done all the required changes in that file

File: src/Components/Home.js (hunk 1 above)
Comment (Yashwant-0108) [reply src/Components/Home.js]: Done

File: src/Components/ProjectCards.js
Comment (Yashwant-0108) [reply src/Components/ProjectCards.js:1]: Deleted the component

File: src/Components/Navbar.js (hunk 2 above)
Comment (anupkore) [reply src/Components/Navbar.js]: Ok

File: src/Components/SignupForm.js
Comment (anupkore) [reply src/Components/SignupForm.js:1]: Ok

**Review by anupkore — APPROVED** [review]

**Pull request merged**